cmd/vela-server/server.go

```diff
@@ -97,7 +100,51 @@ func server(c *cli.Context) error {
 		return err
 	}
 
+	s, err := database.GetSettings(context.Background())
+	if c.Bool("vela-reinitialize-settings-on-startup") || s == nil || err != nil {
+		// only log errors
+		if err != nil {
+			logrus.WithError(err).Error("unable to get platform settings")
+		}
+
+		// create initial settings record
+		s = new(settings.Platform)
+
+		// singleton record ID should always be 1
+		s.SetID(1)
+
+		s.SetCreatedAt(time.Now().UTC().Unix())
```

Comment:
Should we also nullify `updated_by` when this happens?

Reply:
you think empty string or something hardcoded and intentional like `vela-server` ?